runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -413,14 +412,12 @@ public void reconfigureSequencerServers(Layout originalLayout, Layout newLayout,
                         || !originalLayout.getPrimarySequencer()
                         .equals(newLayout.getPrimarySequencer())) {
 
-                    FastObjectLoader fastObjectLoader = new FastObjectLoader(runtime);
-                    fastObjectLoader.setRecoverSequencerMode(true);
-                    fastObjectLoader.setLoadInCache(false);
+                    //TODO(Maithem) why isn't this getting the tails
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)